salt/api-gateway/config/etc-kong-kong.conf

```diff
@@ -54,57 +54,53 @@ anonymous_reports = off          # Send anonymous usage data such as error
 
 
 # modified
-proxy_listen = 0.0.0.0:80     # Address and port on which Kong will accept
+#proxy_listen = 0.0.0.0:80       # Address and port on which Kong will accept
```

Comment:
I've been testing this on Vagrant.
One way you could test this for yourself if you want certainty, is to provision a vagrant box using current formula, checkout branch, run highstate again.